.github/workflows/stale.yml

```diff
@@ -0,0 +1,35 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+    # Run once a day
+    - cron: "30 1 * * *"
+
+jobs:
+  stale:
+    if: "github.repository == 'ESMValGroup/ESMValCore'"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 720
+          days-before-pr-stale: 180
+          days-before-close: 60
+          ascending: true
+          operations-per-run: 300
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 720 days of inactivity.
+
+            If this issue is still important to you, please comment below to remove the stale label. Otherwise, this issue will be automatically closed in 60 days.
+          stale-pr-message: |
+            In order to maintain a backlog of relevant pull requests, we automatically label them as stale after 180 days of inactivity.
+
+            If this pull request is still important to you, please comment below to remove the stale label. Otherwise, this pull request will be automatically closed in 60 days. If this pull request only suffers from a lack of reviewers, please tag the @<span></span>ESMValGroup/technical-lead-development-team so they can help you find a suitable reviewer.
```

Comment:
Why is there an `<span></span>` here? To avoid tagging the team on every stale issue/PR? Does this have the same effect when someone would copy/paste this?

Reply:
Yes exactly, was a tip in the original discussion (https://github.com/ESMValGroup/Community/discussions/238#discussioncomment-13518142) to make a line that could be copy-pasted but doesn't ping automatically.